app/src/main/java/de/xikolo/models/DownloadAsset.kt

```diff
@@ -94,7 +94,7 @@ object DownloadAsset {
             }
 
             class VideoSD(item: de.xikolo.models.Item, val video: Video) :
-                Item(video.streamToPlay.sdUrl, "video_sd_${item.id}.mp4", item) {
+                Item(video.streamToPlay?.sdUrl, "video_sd_${item.id}.mp4", item) {
```

Comment:
Now that I see that the access was safe before, I think it might be better to make `streamToPlay` of type `VideoStream?` again and make an unsafe `!!` call only where there is no other possibility. Sorry for my misjudgement